scRNA-seq-advanced/02-celltype_assignment.Rmd

```diff
@@ -309,284 +310,453 @@ Note that while we applied this technique to assign cell types using the ADT dat
 
 However, what we did here was very ad-hoc and quite manual!
 We didn't calculate any statistics, and we had to look at every tag we were interested in to pick thresholds.
-A different data set might have different background levels, which would require different thresholds. 
+A different dataset might have different background levels, which would require different thresholds. 
 
-While this technique might be good for some simple experiments, and can be useful for manual curation, it might not translate well to more complex data sets with multiple samples.
+While this technique might be good for some simple experiments, and can be useful for manual curation, it might not translate well to more complex datasets with multiple samples.
 We also looked at each marker separately, which might not be the most efficient or robust method of analysis.
 
 For a more principled approach that allows identification of cell types by looking at the expression of sets of genes that are known to characterize each cell type, you might look at the [`AUCell` package](https://bioconductor.org/packages/release/bioc/html/AUCell.html).
-For more on that method, the OSCA chapter [Assigning cell labels from gene sets](http://bioconductor.org/books/3.16/OSCA.basic/cell-type-annotation.html#assigning-cell-labels-from-gene-sets) is a very good reference.
+For more on that method, the OSCA section [Assigning cell labels from gene sets](http://bioconductor.org/books/3.16/OSCA.basic/cell-type-annotation.html#assigning-cell-labels-from-gene-sets) is a very good reference.
 
 
 
 ## Cell type annotation with `SingleR`
 
-First, we need some reference data.
-Text about importance of reference data goes here!
+An alternative approach to using known marker genes for classification is to instead classify cells by way of comparison to a reference dataset.
+To do this, we will find a well-curated gene expression dataset that contains samples with known cell types.
+We can then train a model based on this dataset and look at each of the cells in our new dataset to determine which (if any) of the known cell types has the most similar expression pattern.
+The details of how such a model may be constructed and trained will vary by method, but this overall approach is widely applied. 
```

Comment:
>will vary by method

We probably need to state explicitly that this method is more of the concept "cell annotation by reference" and not a specific method eg singler.

Reply:
Ah! Well then something definitely needs to be clarified since I got a bit lost in antecedents the first time around. Reading it again, I feel like something as simple as "...will vary by specific method" or "will vary by implementation" does it?

Reply:
```suggestion
The details of how such a model may be constructed and trained will vary by the specific method, but this overall approach is widely applied. 
```